crt_portal/static/sass/custom/table.scss

```diff
@@ -1,10 +1,10 @@
 .crt-table {
   @extend .usa-table;
   font-size: 14px;
-  color: $gray-90;
```

Comment:
I believe the text color in the View All table should be `gray-warm-80`, same as the default in rest of the app, as noted here: https://github.com/18F/crt-portal/issues/25.